osquery/core/posix/process.cpp

```diff
@@ -59,7 +59,7 @@ bool PlatformProcess::killGracefully() const {
     return false;
   }
 
-  int status = ::kill(nativeHandle(), SIGINT);
+  int status = ::kill(nativeHandle(), SIGTERM);
```

Comment:
This method should 'ask nicely', if `SIGTERM` causes a crash on macOS or Linux then we should keep `SIGINT`.

Reply:
I tested and this is fine :)